lm_eval/models/huggingface.py

```diff
@@ -90,7 +90,7 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__()
-
+        self.backend = backend
```

Comment:
I think I'd prefer it if we don't set `self.backend` here. Would want our code to expressly error out if a user subclassing HFLM does not ever call `super().__init__()` --> `HFLM._get_backend()` never called.